genai-perf/tests/test_json_exporter.py

```diff
@@ -228,6 +228,7 @@ class TestJsonExporter:
           "output_tokens_mean_deterministic": false,
           "output_tokens_stddev": 0,
           "random_seed": 0,
+          "synthetic_input_files": null,
```

Comment:
`None`?

Reply:
I tried that and it returned an error in the testing (passed when changed to null). It looks like the other parameters that would have None also say null, so I figured it was consistent.

Reply:
I just don't understand how the Python interpreter reads this without an error:

```bash
$ python3.10
Python 3.10.12 (main, Sep 11 2024, 15:47:36) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> null
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'null' is not defined
```

Reply:
Maybe because it's a dictionary value? It looks like we use it throughout the this test.

Reply:
Oh, this is a triple-quoted string, that's a JSON identifier, which is valid.